src/components/EDraggableTitlebar.vue

```diff
@@ -1,6 +1,8 @@
 <script setup lang="ts">
 type Props = {
   title?: string;
+  userCount?: number;
```

Comment:
Usercount feels too much hardcoding, can it be just be a slot with any content? It's ok to TODO / refactor